Resources/Prototypes/Loadouts/Jobs/Medical/traitRestrictedGear.yml

```
@@ -0,0 +1,445 @@
+- type: loadout
+  id: LoadoutMedicalRestrictedGearEyesGlassesChemical
+  category: JobsMedicalChemist
+  cost: 4
+  requirements:
+    - !type:CharacterTraitRequirement
+      traits:
+        - RestrictedGear
+    - !type:CharacterJobRequirement
+      inverted: true
+      jobs:
+         - Chemist
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 8 but found 9

(indentation)